tests/src/CLRTest.Execute.Bash.targets

```diff
@@ -312,7 +277,29 @@ $(BashCLRTestArgPrep)
       <!-- NOTE! semicolons must be escaped with %3B boooo -->
       <_CLRTestExecutionScriptText>
         <![CDATA[
+TakeLock()
```

Comment:
That would be good. Interrupting a build or test run is something I do very often.